azext_edge/edge/providers/edge_api/base.py

```diff
@@ -13,10 +13,11 @@
 
 
 class EdgeResourceApi:
-    def __init__(self, group: str, version: str, moniker: str):
+    def __init__(self, group: str, version: str, moniker: str, label: Optional[str] = None):
         self.group: str = group
         self.version: str = version
         self.moniker: str = moniker
+        self.label: str = label
```

Comment:
type is still Optional[str] since it could be None

Reply:
oh missed that